bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+        except Exception as AuthError:
```

Comment:
Hi, I think this will work, but only for the `bodhi updates new` command. The exception should be managed when the client tries to login.